extcrypto/extcrypto.cc

```diff
@@ -6,12 +6,19 @@
 #include <openssl/pem.h>
 #include <openssl/bio.h>
 #include <openssl/bn.h>
+#include <nan.h>
 
 namespace extcrypto {
 
+  using Nan::Utf8String;
+  using Nan::Callback;
+  using Nan::FunctionCallbackInfo;
+  using Nan::GetFunction;
+  using Nan::New;
+  using Nan::Set;
+
   using v8::Exception;
   using v8::Function;
-  using v8::FunctionCallbackInfo;
   using v8::Isolate;
   using v8::Local;
   using v8::Null;
```

Comment:
Is this still used?

Reply:
Yes, in line 32: `Local<Value> argv[argc] = { Null(isolate), statement };`